### PR TITLE
Rename SabrePublicPluginEvent to BeforeSabrePubliclyLoadedEvent

### PR DESCRIPTION
**Pull request by julien-nc**
And run build/autoloaderchecker.sh

refs #35621 #35784

### CODE REVIEW

File: apps/updatenotification/composer/composer/installed.php [hunk 1]
```diff
@@ -3,7 +3,7 @@
         'name' => '__root__',
         'pretty_version' => 'dev-master',
         'version' => 'dev-master',
-        'reference' => 'e35bcc9381246a8ec3da96e78e9771ef6044a6d4',
```
Comment (ChristophWurst) [inline apps/updatenotification/composer/composer/installed.php:6]: not stricly necessary. composer is a bit weird with those references. IIRC they change every time you commit …

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
New file missing?

File: lib/public/BeforeSabrePubliclyLoadedEvent.php [hunk 2]
```diff
@@ -21,5 +21,5 @@
  */
 namespace OCP;
```
Comment (ChristophWurst) [inline lib/public/BeforeSabrePubliclyLoadedEvent.php:23]:
```suggestion

/**
 * <description of the event>
 *
 * @since 26.0.0
 */
```

**Review by ChristophWurst — CHANGES_REQUESTED** [review]

File: apps/updatenotification/composer/composer/installed.php (hunk 1 above)
Comment (julien-nc) [reply apps/updatenotification/composer/composer/installed.php:6]: Meaning we should rollback those changes?

Comment (ChristophWurst) [reply apps/updatenotification/composer/composer/installed.php:6]: you can but don't have to

**Review by ChristophWurst — APPROVED** [review]

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by nickvergessen** [comment]
Please document in https://github.com/nextcloud/server/issues/34692

**Comment by julien-nc** [comment]
> Please document in #34692

:+1: Done

**Comment by DaphneMuller** [comment]
hello @julien-nc 
Thank you so much for your work on this pull request! This ticket seems to have the tag 'pending documentation'. 
Is there any chance you could clarify what documentation is missing? Is this for admins or for app developers?

**Comment by DaphneMuller** [comment]
Hello!
Thank you for your work on this! I noticed that your PR still has the label 'documentation'. I'm not sure if you already had a look at the documentation for your PR, but at Nextcloud we strive to document changes that affect other app developers or other admins before the release takes place, and I'm pinging you with the hope to get this done. 

Are you familiar with our documentation process already? 
Changes that affect app developers should be documented here:
https://docs.nextcloud.com/server/27/developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_27.html

Changes that affect administrators should be documented here:
https://docs.nextcloud.com/server/latest/admin_manual/release_notes/upgrade_to_27.html

While I understand sometimes it's handy to merge fast to get your changes in, it would be great if next time you could try to add the documentation before merging. You can find more information on that here:
https://docs.nextcloud.com/server/27/developer_manual/prologue/compatibility_app_ecosystem.html#documentation-procedures-of-changes-that-affect-app-developers

If all your documentation efforts are done, please remove the label 'pending documentation' and check the checkbox in your opening note and I'll stop bugging you :) 

Many thanks in advance and thanks again for your work!

**Pull request merged**